<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaValores</class>
 <widget class="QWidget" name="VentanaValores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="leUmbralHueMin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>65</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral hue mín.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral hue máx.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUmbralHueMax">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tam apertura</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTamApertura">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbL2Gradiente">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>L2Gradiente</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbActualizar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar Canny Hue min max apertura L2G</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbProcesar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen procesada</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLineas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar lineas</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRecortada">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen recortada</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BGR to Lab</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contorno (min)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leContornoMin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>120</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRostro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detectar rostro</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHighResolution">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>High resolution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMedianFilter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ksize median filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbMedianFilter">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Median Filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHarris">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Harris Edge detector</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRojoDescartado">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral Rojo descartado</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHSI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HSI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>umbral intensidad HSI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUmbralHSI">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbYUV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>YCbCr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUmbralY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>82</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral Y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHistograma">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>460</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Histograma</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbEliminarSkin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar matiz skin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHmin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHmax">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H max</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbVisualizarHistograma">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Histograma</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDiferenciaVarianzas">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>370</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diferencia varianzas</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUmbralMenosEsto">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral menos esto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
